src/app/Plans/components/PlansTable.tsx

```diff
@@ -172,9 +169,9 @@ const PlansTable: React.FunctionComponent<IPlansTableProps> = ({
     <>
       <Level>
         <LevelItem>
-          <Button variant="secondary" onClick={toggleWizard} isDisabled={false}>
-            Create
-          </Button>
+          <Link to="/planwizard">
```

Comment:
See above re: maybe using `/plans/create` instead of `/planwizard`. Thoughts on that @ibolton336 ?